sycl/include/CL/sycl/detail/image_impl.hpp

```diff
@@ -205,7 +205,9 @@ class __SYCL_EXPORT image_impl final : public SYCLMemObjT {
   }
 
   // Returns the total number of elements in the image
+  __SYCL2020_DEPRECATED("get_count() is deprecated, please use size() instead")
```

Comment:
there is no much sense to deprecate the method in *_impl classes as they are not part of public interface.

Reply:
True that it makes no difference to the end user. However, wouldn't it be useful to change this anyway, just for consistency's sake for the developer?

Reply:
I removed the deprecation warning but kept two methods - get_count() and size()

Reply:
What is the reason for keeping two methods? It is better to remove it now to avoid extra change in future

Reply:
Agreed. Removed get_count().